src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -357,18 +357,28 @@ std::string ContextAttribute::toJson(bool isLastElement)
 {
   std::string  out;
   bool         isNumber = false;
+  bool         isString = true;
 
   if (type == "number")
   {
     isNumber = true;
   }
 
-  if (((type == "") || (isNumber == true)) && (metadataVector.size() == 0))
+  if(type == "string")
```

Comment:
Do we really need special treatment for "string" type? As far as I undersrtand, each type that is not a "number" (or a "date") is by default considered string type from a rendering point of view.